<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConsoleWidget</class>
 <widget class="QWidget" name="ConsoleWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget_co">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="documentMode">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="tab_co_co">
      <attribute name="icon">
       <iconset resource="../OpenRcon.qrc">
        <normaloff>:/icons/console.png</normaloff>:/icons/console.png</iconset>
      </attribute>
      <attribute name="title">
       <string>Console</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_10">
       <item row="1" column="0">
        <widget class="QLineEdit" name="lineEdit_co_co"/>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="pushButton_co_co">
         <property name="text">
          <string>Send</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="QTextEdit" name="textEdit_co_co">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_co_pb">
      <attribute name="icon">
       <iconset resource="../OpenRcon.qrc">
        <normaloff>:/icons/images/icons/punkbuster.png</normaloff>:/icons/images/icons/punkbuster.png</iconset>
      </attribute>
      <attribute name="title">
       <string>PunkBuster</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_11">
       <item row="0" column="0" colspan="2">
        <widget class="QTextEdit" name="textEdit_co_pb">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLineEdit" name="lineEdit_co_pb"/>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="pushButton_co_pb">
         <property name="text">
          <string>Send</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../OpenRcon.qrc"/>
 </resources>
 <connections/>
</ui>
